lua/starfall/instance.lua

```diff
@@ -130,6 +130,10 @@ function SF.Instance.Compile(code, mainfile, player, entity)
 		end
 	end
 
+	instance.whitelistedEnvs = setmetatable({
```

Comment:
This can be changed to `local whitelistedEnvs = {[instance.env] = true}` in builtins.lua above the setfenv/getfenv functions.

Reply:
I can do that, but I think it's better this way in case any other file wants to use `whitelistedEnvs`, or if external Lua wants to modify the table.

Reply:
Sure, that'd be cleaner.

Reply:
Okay, I just moved it over to `builtins.lua` and cached it. I also [wrote a script](https://gist.github.com/x4fx77x4f/e997c64a57389349726e783b8c351554#file-fenvsuicide-lua) to make sure that changing `whitelistedEnvs` externally works, in case there is ever a good reason to do that.